README.md

```diff
@@ -172,155 +214,11 @@ Based on the size and strucutre of the dataset provided + these parameter settin
 | `do_not_balance_hdf5` | bool | False | Include flag for True, no arguments | Include this flag if you DO NOT want to balance HDF5 files, this is not recommended unless the you are dealing with a huge amount of data (many terra bytes), or do not want shuffling between splits. |
 | `keep_split_jsonls` | bool | False | Include flag for True, no arguments | If you DO NOT want to delete split jsonls files that are in text format in the `output_path/splits` directory include this flag. The only reason you would include this flag is if you want to see what text is in each HDF5, meaning that splits/train_1_of_X.jsonl is the jsonl text version of train_1_of_X.hdf5. Including this flag will increase the storage space of your dataset by more than two times. |
 | `num_workers` | int | False | 0 <= `num_workers`<= # of available CPUs | The number of CPU workers to run tokenization with, if the previous run failed due to OOM, you need to decrease this number. |
-</details>
-
-
-## Tokenizing one file
```

Comment:
Reading through this as a first time user confused me a bit, after reading through more critically and jumping around some of the code, I understood the difference. My rationale for moving to the advanced_usage.md file is that we recommend that they use pipeline so in rare cases where a user has pre-defined splits, I have a section below directing them to this info in the `advanced_usage.md` file, but I felt it was a bit overwhelming to keep a whole other table of flags in the main readme that 9/10 users won't be using per our recommendation.